herbert_core/classes/MPIFramework/@ClusterMPI/ClusterMPI.m

```diff
@@ -201,7 +201,11 @@ function check_availability(obj)
                 % future.
                 mpi_exec = fullfile(rootpath,'DLL','_PCWIN64','MS_MPI_R2019b','mpiexec.exe');
             else
+                % use system-defined mpiexec
```

Comment:
you're handling this as PC (I assume Windows) and non-PC ... does the same non-PC configuration work on MacOS, Linux and SCARF deployments?

Reply:
that's Linux build and Mac should be the same. Scarf too if mpiexec is exposed to users. But I think we will need another kind of Cluster, wrapping around qsub or bsub or whatever.